pubsub/README.rst

```diff
@@ -13,54 +13,164 @@ Installation
 Usage
 -----
 
-This Pub/Sub implementation is a slim wrapper around
-``google-cloud-pubsub==0.25.0`` with a few key features:
+This Pub/Sub implementation is based on ``google-cloud-pubsub==0.41.0``
```

Comment:
Could we do anything around version ranges? What's the earliest `google-cloud-pubsub` version which this methodology would support? (I _think_ its 0.26.0, but we'll want to double-check).

Reply:
Looks like we could do `google-cloud-pubsub >= 0.29.4`